contracts-periphery/test/Gas.t.sol

```diff
@@ -0,0 +1,423 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "solmate/test/utils/mocks/MockERC20.sol";
+import "test/utils/DSTestPlus.sol";
+import "src/UmbraBatchSend.sol";
+
+interface UmbraToll {
```

Comment:
[Added](https://github.com/ScopeLift/umbra-protocol/pull/338/files#r882866154) it in the main UmbraBatchSend.sol contract instead since all test contracts testing it will inherit the main contract, is this ok?